src/Turtle/Options.hs

```diff
@@ -0,0 +1,70 @@
+{-# LANGUAGE TypeFamilies #-}
+
+-- TODO: documentation
+
+module Turtle.Options
+    ( Opts.Parser
+    , Opts.InfoMod
+    , options
+    , opt
+    , flag
+    , metavar
+    , helpmsg
+    , shortname
+    , longname
+    , header
+    , footer
+    ) where
+
+import Data.Text (Text)
+import qualified Data.Text as Text
+import Control.Applicative
+import Control.Monad.IO.Class
+import qualified Options.Applicative as Opts
+import qualified Options.Applicative.Types as Opts
+import qualified Options.Applicative.Builder.Internal as Opts
+
+options :: MonadIO io => Opts.Parser a -> Opts.InfoMod a -> io a
```

Comment:
Import the `Parser` type unqualified.  Some versions of haddock will show any qualifiers in the type signature